sources/packages/api/src/route-controllers/student/student.students.controller.e2e-spec.ts

```diff
@@ -60,7 +60,6 @@ describe("Test ATBC Controller", () => {
     simsUser.lastName = faker.name.lastName();
     fakeStudent.user = simsUser;
     const sinValidation = new SINValidation();
-    sinValidation.user = simsUser;
```

Comment:
Why has it started to cause a problem only now?

Reply:
This was caused an issue because delete cascade rules have been removed in the migrations script. I have reverted and fixed this issue by deleting the line of code that was removing/deleting the user object as part of clean up after testing.

Reply:
Implemented code to clean up sin validation after e2e testing.